docs/conf.py

```diff
@@ -300,3 +306,11 @@
 
 # Example configuration for intersphinx: refer to the Python standard library.
 intersphinx_mapping = {'https://docs.python.org/': None}
+
+intersphinx_mapping = {'heat': (
+     'http://f5-openstack-heat.readthedocs.org/en/latest', None),
+     'docs': (
+     'http://f5-openstack-docs.readthedocs.org/en/latest', None),
+     'lbaasv2': (
```

Comment:
This PR is for the lbaasv1 repo, so shouldn't this mapping table point to http://f5-openstack-lbaasv1?